docs/registration.adoc

```diff
@@ -127,4 +127,5 @@ page for the recent Mainnet addresses of the contracts.
 // TODO: Add section with the Keep Network contracts addresses to the Threshold Network docs.
```

Comment:
1. In the line above we could link to the mainnet subpage directly.
2. We can remove this TODO.

Reply:
Fixed in https://github.com/keep-network/keep-core/pull/3381.